segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/admin/commands/ContainerRecoverCommand.java

```diff
@@ -124,21 +123,20 @@ static CommandDescriptor descriptor() {
         private int operationCount = 0;
         private int currentFrameUsedLength = 0;
 
-        private void newOperation(Operation op, List<DataFrame.DataFrameEntry> frameEntries) {
+        private void newOperation(Operation op, List<DataFrameReader.ReadResult.EntryInfo> frameEntries) {
```

Comment:
Inner inner classes being used from the outside is ridiculous. Promote one of those to a top level class.

Reply:
Good point. I extracted this class out and rearranged a few things to avoid the back-and-forth dependencies.